seeds/04_reporting_periods.js

```diff
@@ -1,14 +1,22 @@
 require('dotenv').config()
 
+// NOTE(mbroussard): For now this seed only populates reporting periods for a single tenant with
+// the default tenantId 0 (though you can modify this to manually populate for other tenantIds).
+// In ARPA Reporter-only deployments, there will only be a single tenant anyway so this is fine.
+// Once ARPA Reporter is merged into GOST, there are multiple tenants and when a new tenant is
+// created it will need reporting periods created for it.
+
 // reporting periods loosely based on tables 3 and 4 from here:
 // https://home.treasury.gov/system/files/136/SLFRF-Compliance-and-Reporting-Guidance.pdf
 //
 // when making changes to this file, consider also updating the test seed:
 // tests/server/seeds/07_reporting_periods.js
-exports.seed = async function (knex) {
-  const [{ count }] = await knex('reporting_periods').count('name', { as: 'count' })
+exports.seed = async function (knex, tenantId = 0) {
```

Comment:
It won't get run with any other param by knex natively, but I parameterized this so it's easy to use for populating a new tenant if there's a need to do that in the future. You could either manually hardcode a different value in this one place, or make a different seed file that imports this one and calls with a parameter (potentially multiple times if needed).

I could rewrite this as something like this if you want. It would be slightly safer if e.g. a future version of knex passes multiple arguments of its own.
```
async function seedReportingPeriods(knex, tenantId = 0) {
  // ...
}
exports.seed = knex => seedReportingPeriods(knex)
```